js/ui/shared/ui.editor_factory_mixin.js

```diff
@@ -39,33 +39,35 @@ var EditorFactoryMixin = (function() {
 
     var getTextEditorConfig = function(options) {
         var isValueChanged = false,
-            data = {},
-            isEnterBug = checkEnterBug(),
-            sharedData = options.sharedData || data;
+            isEnterBug = checkEnterBug();
```

Comment:
Rename it, pls. It's not clear what do you check. I'll need to view the whole file to know that you check a current device and browser.